QuickDeployment/QuickDeploymentDesktopReadme.md

```diff
@@ -268,5 +271,41 @@ If you rely on Retail or MAK licensing, you will need to apply the license using
 slmgr.vbs /ipk xxxxx-xxxxx-xxxxx-xxxxx
 ```
 
+___
+## Common Errors And Resolutions
+
+### Unable login to Jenkins website, after browsing to Nexus website
+
+On the QDE VM, once you browse to the Nexus website at `https://chocoserver:8443`, you will receive the following error when trying to browse to the Jenkins website at `http://chocoserver:8080`:
+
+```
+This site can’t provide a secure connection
+ERR_SSL_PROTOCOL_ERROR
+```
+
+This is due to the fact that Nexus has enforced an HSTS policy on the browser, blocking access to unsecured `http:` addresses. This will be fixed in future versions of QDE. In the interim, you can bypass this limitation with the following steps:
+
+1. Add the following line to the file `C:\ProgramData\sonatype-work\nexus3\etc\nexus.properties`:
+```
+jetty.https.stsMaxAge=-1
+```
+2. Close all instances and tabs of Chrome. Open Chrome again, so that only the one tab is open.
+3. Type `chrome://net-internals/#hsts`  into the address bar to access the network internals page.
+4. In the `Delete domain security policies` section near the bottom of the page, type `CHOCOSERVER` (or the hostname if you changed it) and press the `Delete` button.
+5. In the `Query HSTS/PKP domain` field, type `CHOCOSERVER` (or the hostname if you changed it) , and click the `Query` button to confirm that the output is `Not found` (this means the HSTS settings have been removed).
+6. Close all Chrome browser tabs and windows.
+7. Open up an Administrative PowerShell window, and use the following command to restart the Nexus service:
+```powershell
+Restart-Service nexus
+```
```

Comment:
Same here

Reply:
This was the same line as above, so done!